bundles/net.tourbook/src/net/tourbook/data/TourData.java

```diff
@@ -6673,6 +6673,13 @@ public void finalizeTour_SwimData(final TourData tourData, final List<SwimData>
       }
    }
 
+   /**
+    * @return the avgAltitudeChange
```

Comment:
@FJBDev When you reference a field in java doc, start by typing a hash character # then automatically a dropdown is opened with all fields and a link to the field can be inserted.

Reply:
Cool trick, I didn't know.
What I did was to make sure it was like the other get functions like [this one](https://github.com/wolfgang-ch/mytourbook/blob/5ce7c905fdc41333154698bdd903ea721994bd30/bundles/net.tourbook/src/net/tourbook/data/TourData.java#L6684)

`* @return the avgCadence`

Reply:
In the future, I will put this : 
@return {@link #avgAltitudeChange}